cactusbot/handlers/command.py

```diff
@@ -177,9 +179,11 @@ def _modify(self, argument, *modifiers):
 
         for modifier in modifiers:
             if modifier in self.MODIFIERS:
-                argument = self.MODIFIERS[modifier](argument)
-
-        return argument
+                try:
+                    argument = self.MODIFIERS[modifier](argument)
+                    return argument
+                except IndexError:
```

Comment:
We should never be *expecting* an `IndexError`; all modifiers should work, no matter what. (If they don't, we'll just get a logged warning that `CommandHandler` threw an error, and we can fix it accordingly.)